src/main/java/ru/spbstu/calculator/CalculationStrategy.java

```diff
@@ -17,11 +17,14 @@
     public static class Builder {
 
         private double frequencyOfNucleotides = Constants.frequency;
-        private final int sequenceLength;
+        private int sequenceLength = Constants.DEFAULT_WINDOW_SIZE;

```

Comment:
Why you set this to default here? I added this as required because it may be not obvious which window_size we want to use

Reply:
I thought, if we have the constant and anyway use it in `Main`, it would be better to have default PWM and ScoreCalculator constructors. Or are there reasons to make this value choosable every time?